src/qt/sendcoinsdialog.cpp

```diff
@@ -186,6 +187,13 @@ void SendCoinsDialog::setModel(WalletModel *_model)
         // set default rbf checkbox state
         ui->optInRBF->setCheckState(Qt::Checked);
 
+        if (model->privateKeysDisabled()) {
+            ui->sendButton->setText(tr("Create Unsigned"));
+            ui->sendButton->setToolTip(tr("Creates a Partially Signed Bitcoin Transaction (PSBT) for use with e.g. an offline Bitcoin Core wallet, or a PSBT-compatible hardware wallet."));
+        } else {
+            ui->sendButton->setText(tr("Send"));
```

Comment:
[If we do need it,] remove duplicate from .ui file

Reply:
Ok, I removed the tooltip and button text from the UI file. I also added the `s` keyboard shortcut back, though I can't test that on macOS.